tests/integration/test_emergency_contacts.py

```diff
@@ -47,16 +48,22 @@ def test_emergency_contacts_POST(client, auth_headers):
   }
 
   response = client.post(endpoint, json=newContact, headers=auth_headers["admin"])
-  assert is_valid(response, 401) # UNAUTHORIZED - An emergency contact with this name already exists
+  assert is_valid(response, 401) # UNAUTHORIZED - Emergency Contact With This Name Already Exists
+  assert response.json == \
+          {'message':
+           'An emergency contact with this name already exists'}
 
   response = client.post(endpoint, json=newContact, headers=auth_headers["pm"])
   assert is_valid(response, 401) # UNAUTHORIZED - Admin Access Required
+  assert response.json == {'message': 'Admin access required'}
 
   response = client.post(endpoint, json=newContact, headers=auth_headers["pending"]) 
   assert is_valid(response, 401) # UNAUTHORIZED - Admin Access Required
+  assert response.json == {'message': 'Admin access required'}
 
   response = client.post(endpoint, json=newContact)
   assert is_valid(response, 401) # UNAUTHORIZED - Missing Authorization Header
+  assert response.json == {'message': 'Missing authorization header'}
```

Comment:
These extra tests don't really add much value. I will be talking about this topic in the next meeting, and how the test suite is being refactored to avoid redundant tests. They're fine for now, however they will most likely be deleted in the coming months.